src/media.rs

```diff
@@ -51,6 +51,7 @@ impl FromStr for Media {
 
 const TABLE: &[(&str, Media, &[&str])] = &[
   ("application/pdf", Media::Pdf, &["pdf"]),
+  ("application/pgp-signature", Media::Text, &[]),
```

Comment:
```suggestion
  ("application/pgp-signature", Media::Text, &["pgp"]),
```

Reply:
This will fix the CI test that failed. While I've also seen .gpg used, it's usually for encrypted files.